source/codegen/metadata/nidaqmx/enums.py

```diff
@@ -2075,5 +2091,30 @@
                 'value': 10208
             }
         ]
+    },
+    'WriteBasicTEDSOptions': {
+        'values': [
+            {
+                'documentation': {
```

Comment:
You can delete the 'blah' documentation from these.

Reply:
Aw but it's generated!

Yeah, I can fix the tool to filter out any "blah" documentation.

Reply:
Oh yeah, sorry. Forgot you were generating these. Thanks for cleaning it up though!

As a side note, I wonder if we'll ever want to use the documentation in the grpc metadata. Right now I believe we just discard it but we could put it in the generated proto files next to fields. I don't believe proto comments are carried over to generated code but there still might be some value having them in the proto files at least.

Reply:
Yeah, that was borrowed from the original hapigen extraction scripts (which pull it from CVI metadata). Yeah, maybe that's something we could try out at some point. But we'd need to look both at the quality of the result (how is it presented?) and the quality of the documentation itself.

The best-case upside could be pretty significant. Certainly it would be possible for a protoc tool to pull comments into doc strings. But, the actual behavior could be ... blah.

Reply:
Yeah, true. The value to noise ratio would be interesting. Anyway, marking this thread resolved.